frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -316,49 +328,65 @@ export const insightLogic = kea<insightLogicType>([
                         ) {
                             // Instead of making a search for filters, reload the insight via its id if possible.
                             // This makes sure we update the insight's cache key if we get new default filters.
-                            response = await api.get(
+                            ;[response, apiUrl] = await executeInsightRequest(
```

Comment:
You may be avoiding on purpose or not have got there yet but there are also insight interactions in `dashboardLogic.tsx`

* `allItems` includes insight results (if they are in the cache iirc)
* `refreshAllDashboardItems` does the loading of insights when dashboards refresh their insights

Reply:
Intentionally skipped to keep scope of PR down - refactoring those right now :sweat_smile: